lib/gems/pending/VolumeManager/MiqLdm.rb

```diff
@@ -475,7 +475,7 @@ def format(event)
     exit(1)
   end
 
-  parts = disk.getPartitions
+  # parts = disk.getPartitions
```

Comment:
It's part of a `if __FILE__ == $0` section, and I didn't want to mess with the intentions of that (and since it's in there it's relatively benign).  I believe its purpose is to "test" that particular method...I felt removing it hid that.

I'm not sure...what do you think given my reasoning above?